### PR TITLE
17 - Handle info content

### LINKED ISSUE #17: Handling info contents
### Using Markdown for university’s information

- Which lib to use?
- Store them is raw text in database

### Editing university’s information

- Which WYSISWYG editor? (complexity, raw text format)
- Routing? (/exchange/[id]/edit)

### PR DESCRIPTION
**Pull request by tientrinh21**
- Render university information from Markdown
- Add Codespace dev environment

### CODE REVIEW

**Comment by tientrinh21** [comment]
There are lots of commits, feel free to squash if it is too many

File: server/resources_api/universities.py [hunk 1]
```diff
@@ -1,34 +1,55 @@
...
-from database.models import UniversityTable
+from database.models import InfoPageTable, UniversityTable
+
 
 class UniversityRes(Resource):
     @marshal_with(university_resource_fields)
     def get(self, university_id):
-        return db.get_or_404(UniversityTable, university_id,  description=f"No university with the ID '{university_id}'.")
-    
+        sql_raw = "SELECT * FROM university_table JOIN country_table ON university_table.country_code = country_table.country_code WHERE university_table.university_id = :val"
```
Comment (wumbus) [inline server/resources_api/universities.py]: This endpoint has been changed. Swagger needs to be updated

Comment (wumbus) [reply server/resources_api/universities.py]: The raw sql also needs a rewrite

**Review by wumbus — COMMENTED** [review]
The rest looks good

**Pull request merged**